onnxruntime/core/graph/graph.cc

```diff
@@ -4312,7 +4317,7 @@ Status InlineOrCopyInitializer(const Graph& src_graph, const ONNX_NAMESPACE::Ten
 }  // namespace
 
 Status Graph::ProcessSubgraphsInMemoryData(ONNX_NAMESPACE::GraphProto& output_graph_proto,
-                                           bool process_main) const {
+                                           const Graph& main_graph, bool process_main_graph) const {
```

Comment:
When would parent_graph_ be null in a subgraph? There are a number of places like Graph::Resolve where that is the check for it being the main graph, so I'm wondering why that isn't sufficient here.